js/src/components/paid-ads/campaign-preview/mockup-search.js

```diff
@@ -48,7 +48,7 @@ export default function MockupSearch( { product } ) {
 					</ScaledText>
 					<Placeholder stroke="thinner" width="79" color="blue" />
 				</div>
-				<Flex justfy="space-between" align="stretch">
+				<Flex align="stretch">
```

Comment:
The "justfy" is a typo. Removed it because the default value of `justify` is already the "space-between".

Ref: https://github.com/WordPress/gutenberg/blob/%40wordpress/components%4019.17.0/packages/components/src/flex/flex/README.md#justify-csspropertiesjustifycontent